README.md

```diff
@@ -33,11 +36,11 @@ It tests all task parts (generator, checker, solutions and judge).
 ### Task testing overview
 
 What pisek verifies (roughly in order):
- - Samples exist and are not empty
+ - Samples exist
  - The generator generates inputs
     - In Kasiopea mode the generator respects the seed and is deterministic
  - The checker accepts all inputs
-    - It rejects inputs for harder subtasks
+    - If `checker_distinguishes_subtasks`, it rejects inputs for harder subtasks
```

Comment:
It may be clearer if we specify what the value of `checker_distinguishes_subtasks` should be:

```suggestion
    - If `checker_distinguishes_subtasks=on`, it rejects inputs for harder subtasks
```